tests/integrations_app/flagship/test_stable-diffusion.py

```diff
@@ -0,0 +1 @@
+# This is placeholder and the reals file ATM is being copied from the original repo/project
```

Comment:
Consider adding these flagship related files to checkgroup

Reply:
You set it up so `ci/flagship*` triggers the flagship job.

In checkgroup you configure that if `tests/integrations_app/flagship/**` is modified, the flagship job must pass.

This means this works as long as these files are only modified in PRs that follow the branch pattern.

Reply:
I see, lets do it :)